app/controllers/resource_controller.rb

```diff
@@ -103,8 +106,11 @@ def send_to_plastron(id, sparql_update) # rubocop:disable Metrics/MethodLength,
 
       headers = {
         PlastronCommand: 'update',
+        'PlastronArg-on-behalf-of': current_user.cas_directory_id,
         'PlastronArg-no-transactions': 'True',
-        'PlastronArg-recursive': 'False'
+        'PlastronArg-recursive': 'False',
```

Comment:
This header should be removed. The `recursive` argument expects a list of predicate CURIEs (e.g., `pcdm:hasMember,pcdm:hasFile`). Passing the string `'False'` is causing plastron to try to recurse on the blank node `_:False`, which causes several additional HEAD and GET requests.